src/java/org/apache/cassandra/db/filter/RowFilter.java

```diff
@@ -243,6 +245,16 @@ public RowFilter withoutExpressions()
         return withNewExpressions(Collections.emptyList());
     }
 
+    public RowFilter with(Predicate<Expression> filter)
```

Comment:
This name does not appear to match what the method is doing. "With" suggests adding something to the list, while it actually filters stuff out.